api/v1alpha3/openstackcluster_types.go

```diff
@@ -84,9 +84,9 @@ type OpenStackClusterSpec struct {
 	APIServerLoadBalancerAdditionalPorts []int `json:"apiServerLoadBalancerAdditionalPorts,omitempty"`
 
 	// ManagedSecurityGroups defines that kubernetes manages the OpenStack security groups
-	// for now, that means that we'll create two security groups, one allowing SSH
-	// and API access from everywhere, and another one that allows all traffic to/from
-	// machines belonging to that group. In the future, we could make this more flexible.
+	// for now, that means that we'll create a security group allows traffic to/from
```

Comment:
we create two groups for controllers and workers.

Reply:
done